ui/test-view/redux/configureStore.js

```diff
@@ -179,12 +179,13 @@ async function fetchTests(store, fetchParams) {
   // the failed tests.
   // Note: this is in contrast to the "testfailed" entry coming from pushStatus; those are jobs.
   store.dispatch(groupsStore.actions.fetchCounts(pushData.repository.name, getId(pushData.id)));
-  const failedJobs = Object
-    .values(payload.groups)
-    .reduce((gacc, tests) => gacc.concat(Object
-        .values(tests)
-        .reduce((jacc, test) => jacc.concat(test.jobs), []))
-      , []);
+  const failedJobs = Object.values(payload.groups).reduce(
+    (gacc, tests) =>
+      gacc.concat(
+        Object.values(tests).reduce((jacc, test) => jacc.concat(test.jobs), []),
+      ),
+    [],
+  );
```

Comment:
There are several different ways this could be indented/wrapped that would keep ESLint happy, but I went with something similar to what prettier suggested.

Reply:
Sounds good.  Eli had suggested wrapping at the ``.`` on things like this.   I've *sometimes* done that and sometimes not.  But very happy to go with whatever ``prettier`` suggests.  :)